### PR TITLE
Change `lr_min` to `min_lr_factor `

### PR DESCRIPTION
**Pull request by unlimblue**
[fixed] https://github.com/pytorch/torchtitan/issues/1457

### CODE REVIEW

File: torchtitan/models/deepseek_v3/train_configs/deepseek_v3_671b.toml [hunk 1]
```diff
@@ -34,7 +34,7 @@ eps = 1e-8
 warmup_steps = 2_000  # lr scheduler warm up, normally 20% of the train steps
 decay_ratio = 0.8  # lr scheduler decay ratio, 80% of the train steps
 decay_type = "linear"
```
Comment (tianyu-l) [inline torchtitan/models/deepseek_v3/train_configs/deepseek_v3_671b.toml]:
```suggestion
decay_type = "cosine"
```
From DeepSeek V3 paper:
> As for the learning rate scheduling, we first linearly increase it from 0 to 2.2 × 10−4 during the first
2K steps. Then, we keep a constant learning rate of 2.2 × 10−4 until the model consumes 10T
training tokens. Subsequently, we gradually decay the learning rate to 2.2 × 10−5 in 4.3T tokens,
**following a cosine decay curve**. During the training of the final 500B tokens, we keep a constant
learning rate of 2.2 × 10−5 in the first 333B tokens, and switch to another constant learning rate
of 7.3 × 10−6 in the remaining 167B tokens

File: torchtitan/models/deepseek_v3/train_configs/deepseek_v3_16b.toml [hunk 2]
```diff
@@ -34,7 +34,7 @@ eps = 1e-8
 warmup_steps = 200  # lr scheduler warm up, normally 20% of the train steps
 decay_ratio = 0.8  # lr scheduler decay ratio, 80% of the train steps
 decay_type = "linear"
```
Comment (tianyu-l) [inline torchtitan/models/deepseek_v3/train_configs/deepseek_v3_16b.toml]:
```suggestion
decay_type = "cosine"
```

**Review by tianyu-l — APPROVED** [review]
Looks good, thanks!

I think we can use cosine decay in DeepSeek toml configs (following their paper).

**Review by tianyu-l — COMMENTED** [review]
Linting still fails
https://github.com/pytorch/torchtitan/actions/runs/16588102244/job/46967235550?pr=1471

Please fix with
```
pre-commit run --all-files
```
